views/new_contest.ejs

```diff
@@ -10,15 +10,50 @@
 	<link href='https://fonts.googleapis.com/css?family=Sofia' rel='stylesheet'>
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
 	<link rel="stylesheet" type="text/css" href="/css/grid.css">
+  <style>
+	    h1{
+			    font-family: Verdana, Geneva, Tahoma, sans-serif;
+			  	color:border: 1px solid #c59f21;
+          padding:20px;
+          border-radius:5px;
+          background-color: burlywood;
+          color:darkcyan
+			}
+			input[type="text"],
+			input[type="date"],
+			input[type="time"]{
+			    border: 1px solid orange;
+          padding:10px;
+          margin:10px;
+          background-color:honeydew;
+          width:100%;
+			}
+			button[type="submit"]{
+				border: none;
+				background-color: khaki;
+        padding:10px;
+        width:100%;
+        color:lightslategrey
+			}
+			button[type="submit"]:hover{
+          border:none;
+          background-color:#40e068;
+          padding:10px;
+          width:100%;
+          color:lightslategrey
 
+}
+
+
+  </style>
```

Comment:
The styles you added here doesn't look good or match with the Theme.
You could have tried to add colors in the whole inner box and giving a brown background at the heading makes it ugly.

![image](https://user-images.githubusercontent.com/56730716/105142410-93b31400-5b20-11eb-8a49-56ee4f8f3bcf.png)

Reply:
You can try to make it center and instead of using different colors, try using the same color scheme with different contrast.

Yellow somewhere blue somewhere brown doesn't look good.
I hope you get it.